client/indexer-db/src/models/proofs.rs

```diff
@@ -0,0 +1,55 @@
+use diesel::prelude::*;
+use diesel_async::RunQueryDsl;
+
+use crate::{schema::proofs, DbConnection};
+
+/// A single record table that holds the state/metadata of the indexer service.
+#[derive(Debug, Queryable, Insertable, Selectable)]
+#[diesel(table_name = proofs)]
+pub struct Proofs {
```

Comment:
I see we don't keep anything else besides the `last_tick_proven`. Why a new table?
If this is meant only to simplify the abstract `provider_id` (which can be BSP or MSP) - why keeping multiple entries?

Reply:
I added a new table because at first I though we wanted to save all the proofs in the database but indeed now it doesn't make sense.